src/agentlab/agents/generic_agent/agent_configs.py

```diff
@@ -25,8 +26,10 @@
         filter_visible_elements_only=False,
     ),
     action=dp.ActionFlags(
-        multi_actions=False,
-        action_set="bid",
+        action_set=bgym.HighLevelActionSetArgs(
+            subsets=["bid"],
```

Comment:
iiud that a default subset that would get overwritten by the set_benchmark method?